config/oak-cone-detection.json

```diff
@@ -0,0 +1,88 @@
+{
+  "version": 2,
+  "robot": {
+    "modules": {
+      "oak": {
+          "driver": "osgar.drivers.oak_camera:OakCamera",
+          "init": {
+            "fps": 10,
+            "is_color": true,
+            "is_depth": true,
+            "laser_projector_current": 0,
+            "flood_light_current": 500,
+            "is_imu_enabled": true,
+            "number_imu_records": 10,
+            "disable_magnetometer_fusion": false,
+
+            "mono_resolution": "THE_400_P",
+            "color_resolution": "THE_1080_P",
+            "color_manual_focus": 130,
+
+            "stereo_median_filter": "KERNEL_3x3",
+            "stereo_mode": "HIGH_ACCURACY",
+            "stereo_extended_disparity": false,
+            "stereo_subpixel": false,
+            "stereo_left_right_check": true,
+
+            "model": {
+                  "blob": "cone/coneslayer_openvino_2022.1_6shave.blob"
+            },
+            "nn_config": {
```

Comment:
As we discussed yesterday I would move this to a separate file next to model.

Reply:
well, I am not sure yet. There could be more things like reliability threshold, which is application dependent and you do not want to maintain multiple configurations near blob ... for ver1 I would maybe vote now to keep it "to get started" and then rather support some kind of "include" or older "toml" target??

Reply:
It depends how many parameters could be application dependent. These could be here and the default config can be separately...?